sql/util/requests.sql

```diff
@@ -1,10 +1,9 @@
-CREATE TEMPORARY FUNCTION getSummary(payload STRING) -- noqa: PRS
--- SQL Linter expects STRUCT field names to beging with a-z or A-Z so needs noqa ignore command on previous line
```

Comment:
Isn't the `noqa` still needed for fields like `_cdn_provider`?

Reply:
Added this back in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2301/commits/6e551e2c0ad9f44d4c163de9b60bb7e5fcb42799